develop/dev-guide-gui-navicat.md

```diff
@@ -0,0 +1,169 @@
+    - **Password**: enter the password of the TiDB Dedicated cluster.
+
+    ![Navicat: configure connection general panel for TiDB Dedicated](/media/develop/navicat-connection-config-dedicated-general.png)
+
+7. Click the **SSL** tab, check **Use SSL**, **Use authentication** and **Verify server certificate against CA**. Then enter the `<ca_cert>` which you downloaded from TiDB Cloud to **CA Certificate**.
+
+    ![Navicat: configure connection SSL panel for TiDB Dedicated](/media/develop/navicat-connection-config-dedicated-ssl.jpg)
+
+8. **Test Connection** to validate the connection to the TiDB Dedicated cluster.
+
+9. If the connection test is successful, you can see the **Connection Successful** message. Click **Save** to save the connection configuration.
+
+</div>
+<div label="TiDB Self-Hosted">
+
+1. Launch Navicat and click the **Connection** in the upper-left corner. In the selection panel, click **MySQL**  as the database type.
+
+    ![Navicat: add new connection](/media/develop/navicat-add-new-connection.jpg)
+
+2. In the setting pane, configure the following connection parameters:
+
+    - **Connection Name**: give this connection a meaningful name.
+    - **Host**: enter the IP address or domain name of your TiDB Self-Hosted cluster.
+    - **Port**: enter the port number of your TiDB Self-Hosted cluster.
+    - **Username**: enter the username to use to connect to your TiDB.
+    - **Password**: enter the password to use to connect to your TiDB.
+
+    ![Navicat: configure connection general panel for self-hosted TiDB](/media/develop/navicat-connection-config-self-hosted-general.png)
+
+3. Click **Test Connection** to validate the connection to the self-hosted TiDB.
+
+4. If the connection test is successful, you can see the **Connection Successful** message. Click **Save** to finish the connection configuration.
+
+</div>
+</SimpleTab>
+
+## Next steps
+
+- Learn the best practices for TiDB application development with the chapters in the [Developer guide](/develop/dev-guide-overview.md), such as [Insert data](/develop/dev-guide-insert-data.md), [Update data](/develop/dev-guide-update-data.md), [Delete data](/develop/dev-guide-delete-data.md), [Single table reading](/develop/dev-guide-get-data-from-single-table.md), [Transactions](/develop/dev-guide-transaction-overview.md), and [SQL performance optimization](/develop/dev-guide-optimize-sql-overview.md).
+- Learn through the professional [TiDB developer courses](https://www.pingcap.com/education/) and earn [TiDB certifications](https://www.pingcap.com/education/certification/) after passing the exam.
+
+## Need help?
+
+<CustomContent platform="tidb">
+
+Ask questions on the [Discord](https://discord.gg/DQZ2dy3cuc?utm_source=doc), or [create a support ticket](/support.md).
+
+</CustomContent>
+
+<CustomContent platform="tidb-cloud">
+
+Ask questions on the [Discord](https://discord.gg/DQZ2dy3cuc?utm_source=doc), or [create a support ticket](https://support.pingcap.com/).
```

Comment:
@ran-huang This could also point to `/support.md` now, right?

```suggestion
Ask questions on the [Discord](https://discord.gg/DQZ2dy3cuc?utm_source=doc), or [create a support ticket](/support.md).
```

Reply:
Yes.

Reply:
@dveeden @ran-huang For TiDB Cloud, the support doc is: https://docs.pingcap.com/tidbcloud/tidb-cloud-support
```suggestion
Ask questions on the [Discord](https://discord.gg/DQZ2dy3cuc?utm_source=doc), or [create a support ticket](/tidb-cloud/tidb-cloud-support.md).
```